internal/resource_test.go

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) 2023 Cisco Systems, Inc. and its affiliates
 // All rights reserved.
-
```

Comment:
FWIW: here's another spurious line removal.

Keep in mind: there's not a technical problem with removing this blank line.

But it's an extra change that has no purpose in this PR.  So you should probably put that blank line back, just so that this PR **ONLY** has the Artifactory changes -- no spurious changes.

There's at least 2 other places where blank lines were deleted from the tops of `.go` files.  Please put them all back.  Thanks!